kern/openssl.h

```diff
@@ -333,11 +365,36 @@ int probe_entry_SSL_read(struct pt_regs* ctx) {
                               ssl_rbio_num_ptr);
     if (ret) {
         debug_bpf_printk(
-            "(OPENSSL) bpf_probe_read ssl_rbio_num_ptr failed, ret :%d\n",
+            "(OPENSSL) bpf_probe_read ssl_rbio_num_ptr failed, ret: %d\n",
             ret);
         return 0;
     }
 
+    // get ssl->bio->method
+    ssl_rbio_method_ptr = (u64 *)(ssl_rbio_addr + BIO_ST_METHOD);
+    ret = bpf_probe_read_user(&ssl_rbio_method_addr, sizeof(ssl_rbio_method_addr),
+                              ssl_rbio_method_ptr);
+    if (ret) {
+        debug_bpf_printk(
+            "(OPENSSL) bpf_probe_read ssl_rbio_method_ptr failed, ret: %d\n",
+            ret);
+        return 0;
```

Comment:
If the `method` and `type` of the BIO are not required, then I think it is possible to continue with the following process instead of returning 0.

Reply:
those value didn't take effect in SSL_read/SSL_write, we can just make `return 0` to `// return 0`.